Source/Game.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace GameOfLife.Source
+{
+    /// <summary>
+    /// Class for creating new game instance.
+    /// </summary>
+    public class Game
+    {
+        private Field field;
+        private Cells cells;
+        public bool shouldRun = false;
+
+        /// <summary>
+        /// Function that processes game and its logic.
+        /// </summary>
+        public void Play()
+        {
+            const string fieldWidthInfo = "Enter game field width (4 - 120 characters long)";
```

Comment:
move text strings to separate resource file for user info stuff

Reply:
fixed